### DISCUSSION TITLE
protect_from_forgery in Ruby on Rails

### DISCUSSION THREAD

**Discussion by Jennis0928** [community/community#20807] [Other Feature Feedback, Questions, & Ideas] [answered]
What is protect_from_forgery in Ruby on Rails, and why use it?

**Comment by ghost** [comment, marked as answer]
This method is used to prevent csrf attacks.
To protect from any Cross-Site Request Forgery hacker attacks, you need to add 'protect_from_forgery' to your ApplicationController.

**Comment by Jennis0928** [comment]
thank you.
